### PR TITLE
Add rules and cases for fe/lf/af suffixes.  

### PR DESCRIPTION
**Pull request by michaelrhanson**
Specifically handling "oaf" is tricky here since it is an irregular that does _not_ apply in the suffix case (`oaf` to `oafs` but `loaf` to `loaves`).  So I added flag to irregular to require a full-string match.

### CODE REVIEW

File: test/inflector/cases.js [hunk 1]
```diff
@@ -81,7 +81,28 @@
...
+      "elf":"elves",
+      "calf":"calves",
+      "bookshelf":"bookshelves",
+      "wolf":"wolves",
+      "half":"halves",
+      "meatloaf":"meatloaves",
+      "loaf":"loaves",
+      "oaf":"oafs",
+      "jefe":"jefes",
+      "afterlife":"afterlives"
```
Comment (pksunkara) [inline test/inflector/cases.js]: Please add a space after `:`.

Also, can you add the tests for the reverse?

File: lib/defaults.js [hunk 2]
```diff
@@ -58,6 +58,11 @@ module.exports = function (inflect) {
...
   inflect.irregular('cow', 'kine');
   inflect.irregular('zombie', 'zombies');
+  inflect.irregular('oaf', 'oafs', true);
+  inflect.irregular('jefe', 'jefes');
+  inflect.irregular('save', 'saves');
+  inflect.irregular('safe', 'safes');
+  inflect.irregular('fife', 'fifes');
 
-  inflect.uncountable(['equipment', 'information', 'rice', 'money', 'species', 'series', 'fish', 'sheep', 'jeans']);
+  inflect.uncountable(['equipment', 'information', 'rice', 'money', 'species', 'series', 'fish', 'sheep', 'jeans', 'sushi']);
```
Comment (michaelrhanson) [inline lib/defaults.js]: Also slipping this in, see http://www.theatlantic.com/technology/archive/2016/01/whats-the-plural-of-emoji-emojis/422763/ for discussion of noncountability of sushi

**Comment by pksunkara** [comment]
Please comment when this is ready again.

**Comment by michaelrhanson** [comment]
Whitespace fixed; perhaps I misunderstand, but the reverse tests are performed automatically in case `singuralize`?

**Comment by pksunkara** [comment]
> but the reverse tests are performed automatically in case singuralize

You are right. It has been a long time for me :smile:

**Pull request merged**